engine/src/main/java/org/terasology/engine/logic/console/commands/ServerCommands.java

```diff
@@ -68,7 +68,7 @@ public String shutdownServer(@Sender EntityRef sender) {
         EntityRef clientInfo = sender.getComponent(ClientComponent.class).clientInfo;
         DisplayNameComponent name = clientInfo.getComponent(DisplayNameComponent.class);
 
-        logger.info("Shutdown triggered by {}", name.name);
+        logger.atInfo().addArgument(() -> name.name).log("Shutdown triggered by {}");
```

Comment:
simple member access, impact neglectable
```suggestion
        logger.info("Shutdown triggered by {}", name.name); //NOPMD
```